docs/src/getting-started/tutorials/fitting-dists-with-Turing.jl

```diff
@@ -0,0 +1,355 @@
+"""
+
+# ╔═╡ 91279812-9848-48bc-9258-b6f86c9fe923
+md"
+## Fitting a naive model using Turing
+
+We'll start by fitting a naive model using NUTS from `Turing`. We define the model in the `Turing` PPL.
+"
+
+# ╔═╡ a257ce07-efbe-45e1-a8b0-ada40c29de8d
+@model function naive_model(N, y, n)
+    mu ~ Normal(1.0, 1.0)
+    sigma ~ truncated(Normal(0.5, 1.0); lower = 0.0)
+    d = LogNormal(mu, sigma)
+
+    for i in eachindex(y)
+        Turing.@addlogprob! n[i] * logpdf(d, y[i])
+    end
+end
+
+# ╔═╡ 49846128-379c-4c3b-9ec1-567ffa92e079
+md"
+Now lets instantiate this model with data
+"
+
+# ╔═╡ 4cf596f1-0042-4990-8d0a-caa8ba1db0c7
+naive_mdl = naive_model(
+    size(delay_counts, 1),
+    delay_counts.observed_delay .+ 1e-6, # Add a small constant to avoid log(0)
+    delay_counts.n)
+
+# ╔═╡ 71900c43-9f52-474d-adc7-becdc74045da
+md"
+and now let's fit the compiled model.
+"
+
+# ╔═╡ cd26da77-02fb-4b65-bd7b-88060d0c97e8
+naive_fit = sample(naive_mdl, NUTS(), MCMCThreads(), 500, 4)
+
+# ╔═╡ 10278d0c-8c72-4c5f-b857-d3bc6ff2c242
+summarize(naive_fit)
+
+# ╔═╡ 2c0b4f97-5953-497d-bca9-d1aa46c5150b
+let
+    f = pairplot(naive_fit)
+    vlines!(f[1, 1], [meanlog], linewidth = 4)
+    vlines!(f[2, 2], [sdlog], linewidth = 4)
+    f
+end
+
+# ╔═╡ 7122bd53-81f6-4ea5-a024-86fdd7a7207a
+md"
+We see that the model has converged and the diagnostics look good. However, just from the model posterior summary we see that we might not be very happy with the fit. `mu` is smaller than the target $(meanlog) and `sigma` is larger than the target $(sdlog).
+"
+
+# ╔═╡ 080c1bca-afcd-46c0-80b8-1708e8d05ae6
+md"
+## Fitting an improved model using censoring utilities
+
+We'll now fit an improved model by defining our observations as _censored_ using `PrimaryCensored.primarycensored` to construct censored delay distributions from actual delay distributions (which we sample below as `dist`) and uniform primary censored windows (which vary across the data).
+
+Using this approach we can write a log-pmf function `primary_censored_dist_lpmf` that accounts for:
+- The primary and secondary censoring windows, which can vary in length.
+- The effect of right truncation in biasing our observations.
+
+This is the analogous function to the function of the same name in [`primarycensored`](https://primarycensored.epinowcast.org/stan/primarycensored_8stan.html#a40c8992ec6549888fdd011beddf024b0): it calculates the log-probability of the secondary event occurring in the secondary censoring window conditional on the primary event occurring in the primary censoring window by calculating the increase in the CDF over the secondary window and rescaling by the probability of the secondary event occuring within the maximum observation time `D`.
+"
+
+# ╔═╡ f26f6b6b-27f5-4372-b214-d1515c8c0ddc
+function primarycensored_lpmf(dist, y, pwindow, y_upper, D)
```

Comment:
This doesn't make use of the withininterval, or truncate helpers we are designing around so we need a new issue to update.